maxim/logger/livekit/store.py

```
@@ -20,60 +22,41 @@ class SessionState(Enum):
-    temperature: float
-    max_response_output_tokens: Union[str, int]
-    speed: float
-    tools: list
-
-
-class SessionStoreEntry(TypedDict):
+    turn_input_audio_buffer: BytesIO
+    turn_output_audio_buffer: BytesIO
+    trace_id: Optional[str] = None
+
+@dataclass
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)